cordova-lib/src/plugman/fetch.js

```diff
@@ -132,10 +132,14 @@ function fetchPlugin(plugin_src, plugins_dir, options) {
                 ));
             }
             // If not found in local search path, fetch from the registry.
-            var newID = pluginMapperotn[plugin_src];
+            var splitVersion = plugin_src.split('@');
+            var newID = pluginMapperotn[splitVersion[0]];
             if(newID) {
                 events.emit('warn', 'Notice: ' + plugin_src + ' has been automatically converted to ' + newID + ' to be fetched from npm. This is due to our old plugins registry shutting down.');                
```

Comment:
Looks great, thanks @bso-intel!

One small nit - use `splitVersion[0]` in the message instead of `plugin_src` (so, for example, we get `org.apache.cordova.device has been automatically converted to cordova-plugin-device to be fetched from npm` instead of `org.apache.cordova.device@1.1.1 has been automatically converted to cordova-plugin-device to be fetched from npm`).